pydis_site/apps/api/viewsets/bot/infraction.py

```diff
@@ -3,20 +3,15 @@
 from rest_framework.decorators import action
 from rest_framework.exceptions import ValidationError
 from rest_framework.filters import OrderingFilter, SearchFilter
-from rest_framework.mixins import (
-    CreateModelMixin,
-    DestroyModelMixin,
-    ListModelMixin,
-    RetrieveModelMixin
-)
+from rest_framework.mixins import (CreateModelMixin, DestroyModelMixin,
```

Comment:
This doesn’t conform with the style used in the repository, can you please revert it?

Reply:
Sure, I'll update that part also.